doc-Managing_Providers/topics/Amazon_EC2_Manual_Role_Creation.adoc

```diff
@@ -0,0 +1,12 @@
+To eliminate the need to assign _Admin_ group privileges to the Amazon EC2 provider, create an IAM role following the procedure described in link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-service.html[Creating a Role for an AWS Service (Console)] in the Amazon Web Services documentation. 
+
+Use the following parameters:
+
+. Select *EC2* as the service the role will use.
+. Attach the following permissions:
+.. `AmazonEC2FullAccess`
+.. `AmazonS3FullAccess`
+.. `AmazonSQSFullAccess`
+. Input "smartstate" for the *Role name*. 
```

Comment:
Should smartstate be entered with the quote marks? I think it might be more clear with this parameter marked up (with or without quotation marks)